index.html

```diff
@@ -318,210 +318,224 @@ <h4>White a While</h4>
 </section>
 
 <section class="feedback">
-    <div class="container">
-        <div class="row">
-            <div class="col">
-                <div class="feedback__img">
-                    <img src="/img/img_serv.png" alt="">
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="feedback__img">
+          <img src="/img/img_serv.png" alt="">
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div class="feedback__header">
+          <h2>What Our Customers Say</h2>
+          <p>about our community</p>
```

Comment:
I think headers should be inside H2. It looks like both lines are header.
```suggestion
          <h2>What Our Customers Say
            <small>about our community</small>
          </h2>
```